Makefile

```diff
@@ -76,9 +76,14 @@ for-all:
 lint-all:
 	$(MAKE) for-all DIRS="$(PKG_DIRS)" CMD="$(MAKE) lint"
 
+# Needeed temporarily in order to run ci jobs successfully
```

Comment:
Not following these changes that well. What's the goal with these changes? It wouldn't surprise me here if the `checks` rule didn't actually work here. But IIRC it wasn't used in the workflows & needed some additional TLC to make contribution ready.

Reply:
Addressed in live code review and reverted this change.